docs/algob-config.md

```diff
@@ -13,13 +14,15 @@ A network object can specify following entries:
 A special network named `algobchain` can specify the `AlgobChain` configuration:
 
 ```
-  accounts?: NetworkAccounts;
+  accounts_file?: string;
```

Comment:
We already have dashes and camelCaseNames. Do we want to have one more?

Reply:
Or you may want to use `ACCOUNTS_FILE` instead.

Reply:
This is old, we are not using `accounts_file` any more. I will need to change it.